flutter/lib/src/web/sentry_web.dart

```diff
@@ -57,9 +60,36 @@ class SentryWeb with SentryNativeSafeInvoker implements SentryNativeBinding {
   @override
   FutureOr<void> captureEnvelope(
       Uint8List envelopeData, bool containsUnhandledException) {
-    _logNotSupported('capture envelope');
+    _logNotSupported('capture raw envelope data');
   }
 
+  @override
+  FutureOr<void> captureStructuredEnvelope(SentryEnvelope envelope) =>
+      tryCatchAsync('captureEnvelopeObject', () async {
```

Comment:
Do we fail the whole `captureEnvelope` if one item throws?

Reply:
yup, not sure if it makes sense to handle edge cases where we still wanna send them

e.g if an attachment envelope fails and we still send the event that's fine but other way around no event but only an attachment, I guess there are more edge cases like this.

Reply:
Just asked because we do it in the `envelopeStream` method. We should probably also think about revisiting it there if we don't handle these cases.

Reply:
added the implementation that failing items won't stop the processing of sending the envelopes